xlson/__init__.py

```diff
@@ -271,6 +272,66 @@ def create_native_form(survey: Dict) -> Dict:
     return data
 
 
+def generate_metadata() -> Dict:
+    """Create a native form metadata dict."""
+    metadata: Dict[str, Any] = {}
+    inner_attributes = [
+        "start",
+        "end",
+        "today",
+        "deviceid",
+        "subscriberid",
+        "simserial",
+        "phonenumber",
+    ]
+    meta_look_up = {"entity_id": "", "value": ""}
+    for attribute in inner_attributes:
+
+        openmrs_data_type = attribute
+        atrribute_key = "{}".format(attribute)
+        attribute_data = {
+            "openmrs_entity_parent": "",
+            "openmrs_entity": "concept",
+            "openmrs_data_type": openmrs_data_type,
+        }
+
+        # Include openmrs_entity_id attribute
+        # for all inner attributes
+
+        if atrribute_key == "start":
+            attribute_data.update(
+                {"openmrs_entity_id": "163137AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"}
```

Comment:
Is it always the case that `openmrs_entity_id` for the  `start` metadata field always going to be the static value `163137AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA` ?

@dubdabasoduba ^